content/docs/data_registry/installation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Installation Instructions"
+weight: 1
+---
+
+
+# Local FAIR data registry
+
+
+The documentation for the registry is available [here](https://data.scrc.uk/docs/), and is the same for the local and remote registry.
+
+## Installation
+
+There are a few alternative ways to install a local FAIR data registry and we describe the different options below.
+### Dependencies
+
+The registry relies on the [```graphviz```](https://www.graphviz.org) package to produce the schema visualisation, so you will need to follow [graphviz installation process](https://www.graphviz.org/download/) for your system before initialising a local registry.
```

Comment:
Graphviz is also needed for the provenance report visualisation

Reply:
thanks - addressed in https://github.com/FAIRDataPipeline/FDP_hugo/pull/23/commits/58d238b9de923eee0948a9d6b41c6754d38942a3